lib/pengine/native.c

```diff
@@ -541,8 +541,9 @@ add_output_node(GString *s, const char *node, bool have_nodes)
  * \note Caller must free the result with g_free().
  */
 gchar *
-pcmk__native_output_string(pe_resource_t *rsc, const char *name, pe_node_t *node,
-                           uint32_t show_opts, const char *target_role, bool show_nodes)
+pcmk__native_output_string(pe_resource_t *rsc, const char *name,
```

Comment:
Sadly no, uber_parent() is in the call chain, and while that seems like it could be const, it can return rsc itself if there is no parent, and the return value cannot be const. We could create a parallel version of it that returns a const pointer, and I might do that at some point, but not now